python/cudaq/kernel/ast_bridge.py

```diff
@@ -1364,6 +1364,19 @@ def bodyBuilder(iterVal):
                 opCtor([], [], [], [qubitA, qubitB])
                 return
 
+            if node.func.id == 'u3':
+                # Single target, three parameters `u3(θ,φ,λ)`
+                numValues = len(self.valueStack)
+                target = self.popValue()
+                params = [self.popValue() for _ in range(numValues - 1)]
+                params.reverse()
```

Comment:
Not a big deal, but we expect exactly 3 angles here so it would be straightforward to create the params without the `reverse()`.

```python
lambd4 = self.popValue()
phi = self.popValue()
theta = self.popValue()
params = [theta, phi, lambd4]
```

I assume there is no way to pop a vector of values in natural order like in the C++ bridge.

```c++
std::vector<Value> args = popValues(3);
```